components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -148,7 +193,11 @@ define([
 
             console.log("Export data: >>" + exportString + "<<");
             if (exportString != "") {
-                FileSaver.saveTextAs(exportString, fileName);
+                if (exportType != "image") {
+                    FileSaver.saveTextAs(exportString, fileName);
+                } else if (exportType == "image" && imageType == "svg") {
+                    FileSaver.saveTextAs(exportString, fileName, 'base64', 'data:image/png;base64');
```

Comment:
it's not "SaveAsText". It is "saveTextAs". This line is a Safari fix, in this case we saving text as 'data:image/png'.